README.md

```diff
@@ -1 +1 @@
-# pseudoanonymization-service
```

Comment:
Add at least the following here:

1. A description of what the service is / does.
2. How to set up for development (installing packages, running tests, setting up pre-commit hooks, running the service locally, how to make a test request etc.)

See the backend or app repo for an example